website/src/pages/BlogPage/style.ts

```diff
@@ -165,24 +165,58 @@ const useStyles = makeStyles((theme: Theme) => ({
       color: theme.palette.text.secondary,
       lineHeight: '22px'
     },
-    '& ul li':{
-      fontSize: '1rem',
-      color: theme.palette.text.secondary,
-      lineHeight: '22px'
+    '& ul, & ol':{
+      paddingLeft: 0,
+      '& li':{
+          paddingLeft: theme.spacing(1.85),
+          fontSize: '1rem',
+          color: theme.palette.text.secondary,
+          lineHeight: '22px',
+          listStyle: 'none',
+          position: 'relative',
+      },
+      '& li:not(:last-child)':{
+        marginBottom: theme.spacing(1)
+      },
+    },
+    '& ul': {
+      '& li': {
+        '&::before':{
+          content: '""',
+          position: "absolute",
+          left: 0,
+          top: '6px',
+          width: '8px',
+          height: '8px',
+          borderRadius: '100%',
+          background: theme.palette.info.light
+        }
+      }
     },
     '& h2': {
       fontSize: '2rem', 
       lineHeight: '50px'
     },
-    '& code':{
-      overflow: 'auto',
+    '& pre code':{
       borderRadius: '4px',
       display: 'block',
       overflowX: 'auto',
       padding: '1em',
-      background: theme.palette.grayishGreen.dark,
+      background: theme.palette.primary.dark,
```

Comment:
why this color change?

Reply:
The existing color is as per figma screen https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=4%3A6260

Reply:
That's a screenshot,

Reply:
https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=981%3A479 the code block taken from documents design

Reply:
Correct, but I think code block screenshot is added with the intended color theme.

Reply:
i dont know but the same screenshot is present in current blog also, https://openebs.io/blog/openebs-node-device-management-ndm-e2-80-8a--e2-80-8atroubleshooting-tips/

Reply:
i think its better to keep the same style between docs and website

Reply:
Okay